drivers/remote/driver.go

```diff
@@ -13,28 +13,11 @@ var errNoCallback = errors.New("No Callback handler registered with Driver")
 const remoteNetworkType = "remote"
 
 type driver struct {
-	networkType string
-	callback    driverapi.DriverCallback
```

Comment:
Isnt it true that the callback must be retained by the in-built `Remote`-integration driver ?
Not necessarily as part of the object, but somewhere which can be used when the actual remote driver is detected ?
If yes, can you please add some code to make sure we have proper alternative in place so that readers understand the design intent ?